oaknational/Oak-Web-Application
# CTC-92: Lesson overview vocab button component

## Description

Music year: 1966

- Adds `LessonOverviewVocabButton` component to `LessonDetails` component
- Updates `LessonDetails` test file
- onClick should navigate to additional materials secion

ACs:

- [ ]  Add title and button to the top of the lesson details section for MFL lessons (small button component)
- [ ]  onClick this button navigates to the additional materials section - same scrolling behaviour as side menu
- [ ]  Only display button if lesson has additional materials

## Issue(s)

Fixes #CTC-92

## How to test

1. Go to https://deploy-preview-3053--oak-web-application.netlify.thenational.academy/teachers/beta/lessons/des-auteurs-francophones-perfect-tense-with-etre - hard coded for this lesson as fixture data being utilised 
2. Button should render in the lesson details component, not displayed on non-beta pages
3. When clicking the button, screen scrolls to additional material section of the page

## Screenshots

How it should now look:

<img width="908" alt="Screenshot 2024-12-05 at 01 07 31" src="https://github.com/user-attachments/assets/64e88ed6-f7cb-4e2b-b8b0-f2ac5e8dfa29">

https://github.com/user-attachments/assets/39db8445-d4f8-4108-b55e-2ba529333d2f


## Checklist
